mllib/src/main/scala/org/apache/spark/ml/feature/LSH.scala

```diff
@@ -71,6 +72,10 @@ private[ml] abstract class LSHModel[T <: LSHModel[T]]
   /** @group setParam */
   def setOutputCol(value: String): this.type = set(outputCol, value)
 
+  /** @group expertSetParam */
+  @Since("3.0.0")
+  def setRelativeError(value: Double): this.type = set(relativeError, value)
```

Comment:
I agree with @srowen. Seems this relative error does not mean too much for users in LSH. If they want more or less NN, they can change numNearestNeighbors.